src/utils/convertCases/convertCases.ts

```diff
@@ -1,8 +1,14 @@
-export function convertCases(add: string): string {
-    return add
-        .replace('+2', '+Gen.')
-        .replace('+3', '+Dat.')
-        .replace('+4', '+Acc.')
-        .replace('+5', '+Ins.')
-        .replace('+6', '+Loc.');
+export function convertCases(caseInfoRaw: string): string {
+    const caseList = ['+Nom','+Gen','+Dat','+Acc','+Ins','+Loc','+Voc'];
```

Comment:
Please move outside. I am not 100% sure if this is optimized and the array is not being created each time we enter the function.